qa/rgw/store/sfs/tests/test-sfs-lifecycle-smoke.py

```diff
@@ -0,0 +1,274 @@
+        letters = string.ascii_lowercase
+        result_str = "".join(random.choice(letters) for i in range(length))
+        return result_str
+
+    def get_random_bucket_name(self) -> str:
+        return self.get_random_name(LifecycleSmokeTests.BUCKET_NAME_LENGTH)
+
+    def get_random_object_name(self) -> str:
+        return self.get_random_name(LifecycleSmokeTests.OBJECT_NAME_LENGTH)
+
+    def generate_random_file(self, path, size=4):
+        # size passed is in mb
+        size = size * 1024 * 1024
+        with open(path, "wb") as fout:
+            fout.write(os.urandom(size))
+
+    def assert_bucket_exists(self, bucket_name):
+        response = self.s3_client.list_buckets()
+        found = False
+        for bucket in response["Buckets"]:
+            if bucket["Name"] == bucket_name:
+                found = True
+        self.assertTrue(found)
+
+    def upload_file_and_check(self, bucket_name, object_name):
+        test_file_path = os.path.join(self.test_dir.name, object_name.replace("/", "_"))
+        self.generate_random_file(test_file_path)
+        # upload the file
+        self.s3_client.upload_file(test_file_path, bucket_name, object_name)
+
+        # get the file and compare with the original
+        test_file_path_check = os.path.join(self.test_dir.name, "test_file_check.bin")
+        self.s3_client.download_file(bucket_name, object_name, test_file_path_check)
+        self.assertTrue(
+            filecmp.cmp(test_file_path, test_file_path_check, shallow=False)
+        )
+
+    def check_objects_exist(self, objects, objects_expected):
+        objects_found = 0
+        for obj in objects:
+            if obj["Key"] in objects_expected:
+                objects_found += 1
+        self.assertEqual(len(objects_expected), objects_found)
+        self.assertEqual(len(objects_expected), len(objects))
+
+    def create_random_buckets(self, num_buckets):
+        buckets = []
+        for i in range(num_buckets):
+            bucket_name = self.get_random_bucket_name()
+            buckets.append(bucket_name)
+            self.s3_client.create_bucket(Bucket=bucket_name)
+            self.assert_bucket_exists(bucket_name)
+        return buckets
+
+    def test_expiration(self):
+        bucket_name = self.get_random_bucket_name()
+        self.s3_client.create_bucket(Bucket=bucket_name)
+        self.assert_bucket_exists(bucket_name)
+        objects = [
+            "expire1/foo",
+            "expire1/bar",
+            "keep2/foo",
+            "keep2/bar",
+            "expire3/foo",
+            "expire3/bar",
+        ]
+        for obj in objects:
+            self.upload_file_and_check(bucket_name, obj)
+
+        response = self.s3_client.list_objects(Bucket=bucket_name)
+        self.check_objects_exist(response["Contents"], objects)
+
+        rules = [
+            {
+                "ID": "rule1",
+                "Expiration": {"Days": 1},
+                "Prefix": "expire1/",
```

Comment:
According to the [AWS docs](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#getBucketLifecycleConfiguration-property), the `Prefix` is not used anymore.
```
Prefix identifying one or more objects to which the rule applies. This is no longer used; use Filter instead.
```
This should look like 
```
"Filter": {
   "Prefix": "expire1/"
},
```
according to the [docs](https://docs.aws.amazon.com/AmazonS3/latest/userguide/how-to-set-lifecycle-configuration-intro.html).